### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
Added Modal and Grid

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Both code and UI is good. you can merge it

File: Componentss/image.html [hunk 1]
```diff
@@ -1,169 +1,207 @@
...
+            class="resp-img"
+            src="https://5.imimg.com/data5/SELLER/Default/2020/9/SX/XX/AC/11037343/whatsapp-image-2020-09-08-at-1-54-23-pm-500x500.jpeg"
+            alt="responsive-image"
+          />
+        </div>
+
+        <!-- Code Snippet -->
+        <iframe
+          src="https://www.thiscodeworks.com/embed/61fb33e66ab14d0015c32cc1"
+          style="width: 100%; height: 99px"
```
Comment (aayushjain-code) [inline Componentss/image.html:201]: try to give class for specific repeated iframes instead of giving inline styling again and again

**Review by aayushjain-code — APPROVED** [review]

**Comment by MuskanChhatrasal** [comment]
Done with Modal and Grid

File: components/Modal.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.modal-cont {
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    padding-top: 8rem;
+    position: fixed;
+    z-index: 10;
+    background-color: #00000066;
```
Comment (aayushjain-code) [inline components/Modal.css:9]: try to get the colors from root directory

**Pull request merged**